aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -1617,6 +1699,10 @@ func resourceAviatrixTransitGatewayUpdate(d *schema.ResourceData, meta interface
 			}
 			newHaGwEnabled = true
 		} else if deleteHaGw {
+			if d.Get("ha_gw_size").(string) != "" {
+				return fmt.Errorf("\"ha_gw_size\" must be empty if transit HA gateway is deleted")
+			}
+
```

Comment:
Same as Spoke gw comment above, should we check the other `ha_*` attributes as well?